### DISCUSSION TITLE
How to use $and and $or

### DISCUSSION THREAD

**Discussion by wisac** [ppetzold/nestjs-paginate#995] [Q&A]
How do i achieve this with filter: `(A=1 && B=2) OR (C=3 && D=4)`.
I can't seem to know how write this query filter. 
Thanks.

**Comment by Helveg** [comment]
There is currently no support for multiple grouped comparators across columns. The only multi-comparators we support are on the same column, such as: `(A LIKE mike && A LIKE walks) OR (A LIKE ethan && A LIKE runs)`..

Filters between different columns are always comined as a collection of AND-joined siblings. I think!
